sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/LineEnding.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.documents.indexes.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/** Defines values for LineEnding. */
+public final class LineEnding extends ExpandableStringEnum<LineEnding> {
```

Comment:
Immutable annotation?

Reply:
Are these files code generated?

Reply:
Yes, this is a code generated class, not sure why this is missing the annotation.

@jianghaolu, @srnagar do you know why this is missing the annotation?